posts/2023-10-17-23.0.0.10.adoc

```diff
@@ -0,0 +1,162 @@
+
+
+[#run]
+
+== Develop and run your apps using 23.0.0.10
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], include the following in your `pom.xml` file:
+
+[source,xml]
+----
+<plugin>
+    <groupId>io.openliberty.tools</groupId>
+    <artifactId>liberty-maven-plugin</artifactId>
+    <version>3.8.2</version>
+</plugin>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle], include the following in your `build.gradle` file:
+
+[source,gradle]
+----
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.6.2'
+    }
+}
+apply plugin: 'liberty'
+----
+
+Or if you're using link:{url-prefix}/docs/latest/container-images.html[container images]:
+
+[source]
+----
+FROM icr.io/appcafe/open-liberty
+----
+
+Or take a look at our link:{url-prefix}/start/[Downloads page].
+
+If you're using link:https://plugins.jetbrains.com/plugin/14856-liberty-tools[IntelliJ IDEA], link:https://marketplace.visualstudio.com/items?itemName=Open-Liberty.liberty-dev-vscode-ext[Visual Studio Code], or link:https://marketplace.eclipse.org/content/liberty-tools[Eclipse IDE], try our open source link:https://openliberty.io/docs/latest/develop-liberty-tools.html[Liberty developer tools] for efficient development, testing, debugging, and application management, all within your IDE. 
+
+[link=https://stackoverflow.com/tags/open-liberty]
+image::img/blog/blog_btn_stack.svg[Ask a question on Stack Overflow, align="center"]
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/26423
+// Contact/Reviewer: gjwatts
+// // // // // // // // 
+[#java21]
+== Support for Java 21
+// The following excerpt for issue https://github.com/OpenLiberty/open-liberty/issues/26192 was found in 2023-09-26-23.0.0.10-beta.adoc.
+// ------ <Excerpt From Previous Post: Start> ------
+// Contact/Reviewer: gjwatts
+// // // // // // // // 
+
+Java 21 is finally here, the first long-term support (LTS) release since Java 17 was released two years ago. It offers some new functionality and changes that you'll want to check out for yourself.
+
+Take advantage of trying out the new changes in Java 21 now and to test out your applications, microservices, and runtime environments.
+
+To run Open Liberty with Java 21:
+
+1. Download and install link:https://openliberty.io/start/#runtime_releases[Open Liberty 23.0.0.10], or later.
+2. Download the latest release of Java 21 from link:https://adoptium.net/temurin/releases/?version=21[adoptium.net].
+3. Edit your Open Liberty runtime link:/docs/latest/reference/config/server-configuration-overview.html#server-env[server.env file] to point `JAVA_HOME` to your Java 21 installation.
+4. Start testing!
+
+Here are some highlights from new JEP changes in Java 18-21:
```

Comment:
what does "18-21" mean here? Changes between Java 18 and 21?

Reply:
Correct, changes between Java 18 and 21.